### PR TITLE
Fix the error of not inserting orderid into the orders 

### PR DESCRIPTION
**Pull request by r05921104**
Fix the errors of not inserting orderid into orders when uploading order files.

### CODE REVIEW

File: routes/uploader.js [hunk 1]
```diff
@@ -42,17 +42,43 @@ router.post('/uploader', function (req, res) {
...
-          logger.log('All items have been done')
-          res.json({
-            state: 'success',
-            state_array: itemStateArray
-          })
-        }
+        sql = "select * from orders where orderid='" + orderid + "'"
+        dbclient.query(sql, function (err, result) {
+          if (err) {
+            console.log('select orders err')
```
Comment (tz70s) [inline routes/uploader.js]: Use `logger` here.

File: static/js/uploader.js [hunk 2]
```diff
@@ -14,8 +14,9 @@ $(document).ready(function () {
       success: function (data) {
         var state_array = data['state_array']
         for (var i = 0; i < state_array.length; i++) {
-          var row = i + 1
-          $('#upload_status' + row).val(state_array[row - 1]['upload_state'])
+          var row = state_array[i]['row']
+          var state  = state_array[i]['upload_state']
+          $('#upload_status' + row).val(state)
```
Comment (tz70s) [inline static/js/uploader.js]: `var` -> `let`?

File: routes/uploader.js (hunk 1 above)
Comment (r05921104) [reply routes/uploader.js]: I forgot to replace it.

File: static/js/uploader.js (hunk 2 above)
Comment (r05921104) [reply static/js/uploader.js]: ok

**Comment by tz70s** [comment]
After merge this, it’s better write a logger.err for err redirection.

**Pull request merged**